code/src/layouts/header/components/header-toolbar/HeaderToolbar.scss

```diff
@@ -0,0 +1,31 @@
+@import "@design-system";
+
+.header__wrapper {
+  padding: spacing(3) spacing(4);
+  background-color: var(--spa-wrapper-background-color, $white);
```

Comment:
It would be nicer to use some light background
for example like our root background $ghost-alpha
On mock-up it is white because have a dark space above

![image](https://github.com/idx-academy/spa-orders/assets/127191734/5a1f4649-6c3d-4040-9385-99220c0868d1)

Reply:
I can't use ghost-white because it adds alpha channel and makes header transparent. Maybe let's keep it like it is now.

![image](https://github.com/idx-academy/spa-orders/assets/109922101/690d201a-03b9-424c-96dc-6b19d4cf4a2c)